src/Nox.Generator/Domain/Entity.template.cs

```diff
@@ -10,25 +10,26 @@
 using Nox.Types;
 
 namespace {{codeGeneratorState.DomainNameSpace}};
-internal partial class {{className}}:{{className}}Base
+
+internal partial class {{className}} : {{className}}Base
 {
 
 }
-{{- if entity.Persistence.Create.RaiseEvents }}
+{{- if entity.Persistence.Create.RaiseEvents == "DomainEventsOnly" || entity.Persistence.Create.RaiseEvents == "DomainAndIntegrationEvents" }}
```

Comment:
on the EntityCreateSettings (Nox Solution Yaml definition)
we could add something like this:

```
    [YamlIgnore]
    public bool RaiseDomainEvents { get; internal set; } => RaiseEvents == RaiseEventsType.DomainEventsOnly
```

This would be more meaningful, strong and compiled, and on scriban the usage would be more simple

if you agree do also for Edit and Delete

Reply:
Yes, agreed. Changes have been pushed.